keyboards/gami_studio/lex60/lex60.h

```diff
@@ -19,3 +19,49 @@
     {k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, XXX}, \
     {k40, k41, k42, XXX, XXX, k45, k46, XXX, XXX, XXX, k4a, k4b, k4c, k4d, XXX}  \
 }
+
+#define LAYOUT_uhhkb( \
+    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d,      \
+    k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b,      k2d,      \
+    k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c,           \
+         k41, k42,           k45, k46,           k49, k4a, k4b                 \
+) \
+{ \
+    {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e}, \
+    {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, XXX}, \
+    {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, XXX, k2d, XXX}, \
+    {k30, XXX, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, XXX, XXX}, \
+    {XXX, k41, k42, XXX, XXX, k45, k46, XXX, XXX, k49, k4a, k4b, XXX, XXX, XXX}  \
+}
+
+#define LAYOUT_hhkb( \
```

Comment:
HHKB layout is one of our Community Layouts.

[LAYOUT_60_hhkb](https://github.com/qmk/qmk_firmware/tree/abd36de5adbb484695d3500ad790df557d4f5419/layouts/default/60_hhkb)

```suggestion
#define LAYOUT_60_hhkb( \
```

Reply:
Is this gonna work even when there are extra pins when compared to the commonly available layout?